File: C++/Practice Q/Day2/stack_implementation.cpp
#include <iostream>
#include <stack>
using namespace std;

int main() {
    stack<int> s;  // Create a stack

    s.push(10);    // Push elements
    s.push(20);

    s.pop();       // Pop top element

    cout << "Top element: " << s.top() << endl;

    return 0;
}

File: Practice Q/Day2/Java/stack_implementation.java
import java.util.Stack;

public class Main {
    public static void main(String[] args) {
        Stack<Integer> s = new Stack<>();

        s.push(10);
        s.push(20);
        s.pop();

        System.out.println("Top element: " + s.peek());
    }
}
